### PR TITLE
[News] 2020-08-04

### PR DESCRIPTION
**Pull request by ybiquitous**
See https://github.com/sider/runners/blob/0.32.0/CHANGELOG.md

### CODE REVIEW

File: docs/getting-started/custom-configuration.md [hunk 1]
```diff
@@ -228,6 +228,28 @@ When the `npm_install` option is not `false`, Sider will try as follows:
 When the option is `false`, Sider will skip these installation steps and analyze with the pre-installed default version.
 You might want to set the option to `false` if you don't configure analyzer and don't want to see warnings.
 
+## `linter.<analyzer_id>.jvm_deps`
+
+_Type:_ `string[]`
+
+For JVM tools such as Checkstyle or ktlint, this option allows you to load 3rd-party rules or plugins. For example:
```
Comment (yykamei) [inline docs/getting-started/custom-configuration.md:235]: I think "third-party" is more common.

```suggestion
For JVM tools such as Checkstyle or ktlint, this option allows you to load third-party rules or plugins. For example:
```

File: docs/news/2020.md [hunk 2]
```diff
@@ -11,6 +11,44 @@ This document describes notable changes on Sider in the year 2020.
 
 ---
 
+## Support 3rd-party dependencies with JVM tools
+
+<time class="news-date" datetime="2020-08-04">August 4, 2020</time>
+
+We have just supported 3rd-party dependencies with JVM tools in this release!
+This feature allows you to load 3rd-party rules or plugins via `sider.yml`. For example:
```
Comment (yykamei) [inline docs/news/2020.md:19]: I'm not sure which is better, `for` and `with`, but I feel `for` is better for me.

```suggestion
We have just supported third-party dependencies for JVM tools in this release!
This feature allows you to load third-party rules or plugins via `sider.yml`. For example:
```

File: docs/tools/kotlin/ktlint.md [hunk 3]
```diff
@@ -19,95 +19,59 @@ hide_title: true
...
-| [`gradle.output`](#gradleoutput)                                                      | `string`                            | -            |
-| [`maven`](#maven)                                                                     | `hash`                              | -            |
-| [`maven.goal`](#mavengoal)                                                            | `string`                            | _(required)_ |
-| [`maven.reporter`](#mavenreporter)                                                    | `"checkstyle"`, `"json"`, `"plain"` | _(required)_ |
-| [`maven.output`](#mavenoutput)                                                        | `string`                            | _(required)_ |
+---
 
 ### `cli`
 
+> This option was **removed**.
```
Comment (yykamei) [inline docs/tools/kotlin/ktlint.md:73]: Let's delete the removed options in the next release.

Comment (ybiquitous) [reply docs/tools/kotlin/ktlint.md:73]: I will keep them for a while.

**Comment by ybiquitous** [comment]
Thanks for the review!

**Pull request merged**